public/app/features/dashboard/components/PanelEditor/VisualizationSelectPane.tsx

```diff
@@ -28,10 +29,15 @@ interface Props {
 export const VisualizationSelectPane = ({ panel, data }: Props) => {
   const plugin = useSelector(getPanelPluginWithFallback(panel.type));
   const [searchQuery, setSearchQuery] = useState('');
-  const [listMode, setListMode] = useLocalStorage(
-    LS_VISUALIZATION_SELECT_TAB_KEY,
-    VisualizationSelectPaneTab.Visualizations
-  );
+
+  // Add support to show widgets in the visualization picker
+  const isWidget = !!plugin.meta.skipDataQuery;
```

Comment:
```suggestion
  const isWidget = isWidget(plugin);
```
I'd try to extract this logic to an external function to be reused around.

Reply:
For now, I would prefer to have this here :).